### PR TITLE
Ft email verification 160583995

### PR DESCRIPTION
**Pull request by nakimera**
**What does this PR do?**
This PR sends a user an email upon successful registration using SendGrid. 

**Description of the task to be completed**
When a user successfully registers, they receive a verification link via the email they used on registration. It is after activating their account that they can login.

**How should this be tested?**

After cloning the repo, checkout ft-email-verification-160583995 branch and run the server.
Register a user with a valid(your working) email address, check your email for a link to activate your account. 

**Screenshots** 

Successful registration

screen shot 2018-10-08 at 20 26 56

Successful activation

screen shot 2018-10-15 at 13 58 59

Run tests

screen shot 2018-10-08 at 20 27 43

#160583995

### CODE REVIEW

**Comment by JoyLubega** [comment]
Fix the travis Error

**Comment by JoyLubega** [comment]
Fix the conflicts on this PR

**Review by JoyLubega — CHANGES_REQUESTED** [review]
"message": "Your account has been successfully activated. Please Login to complete profile"
I don't think you should return this message. Because a user gets a token even after registration. After verification the user doen't need to login

File: authors/apps/authentication/serializers.py [hunk 1]
```diff
@@ -22,7 +22,7 @@ class Meta:
         model = User
         # List all of the fields that could possibly be included in a request
         # or response, including fields specified explicitly above.
-        fields = ['email', 'username', 'password','token','complete_profile']
```
Comment (j0flintking02) [inline authors/apps/authentication/serializers.py:25]: hey @nakimera thanks for the great work i think this line was meant to be there don't you think

Comment (nakimera) [reply authors/apps/authentication/serializers.py:25]: hey @j0flintking02. Thanks for the feedback. The line is actually still there. I just modified it by removing the complete profile field. A user can only use the app after activating their account. This includes completing their profile.  They get the prompt to complete their profile after activating the account.

Comment (nakimera) [reply authors/apps/authentication/serializers.py]: hey @JoyLubega. Thanks for pointing out that the user doesnot need to login. I had missed that. I however think it is necessary to prompt them that their accout has been activated  and they need to complete their profile.

**Comment by nakimera** [comment]
hey @JoyLubega. Thanks for pointing out that the user does not need to log in. I had missed that. I, however, think it is necessary to prompt them that their account has been activated and they need to complete their profile.

**Comment by JoyLubega** [comment]
Yes, they should get a message informing them about updating there profile..

**Review by JoyLubega — APPROVED** [review]
Good work!!!

**Pull request merged**